content/events/clean-architecture-workshop.mdx

```diff
@@ -0,0 +1,117 @@
+---
+eventHeader:
+  heroBackground: /images/background/polygonBackground.png
+  altText: Clean Architecture
+  imgOverlay: /images/background/CleanArchitecture-Workshop.png
+footer: ''
+seo:
+  title: The Clean Architecture 2-day Workshop
+  description: >-
+    The best approach to building enterprise applications using Clean
+    Architecture and .NET
+_body:
+  - eventDurationInDays: 2
+    price: 440
+    eventList:
+      - city: Brisbane
+        date: 2023-07-23T14:00:00.000Z
+        bookingURL: >-
+          https://www.eventbrite.com.au/e/clean-architecture-2-day-workshop-brisbane-tickets-632508980387
+      - city: Melbourne
+        date: 2023-07-31T14:00:00.000Z
+        bookingURL: >-
+          https://www.eventbrite.com.au/e/clean-architecture-2-day-workshop-melbourne-tickets-632510504947
+      - city: Sydney + Online
+        date: 2023-08-06T14:00:00.000Z
+        bookingURL: >-
+          https://www.eventbrite.com.au/e/clean-architecture-2-day-workshop-sydney-online-tickets-632511417677
+    _template: EventBooking
+  - header: Presenters
+    presenterList:
+      - presenter: content/presenters/daniel-mackay.mdx
+      - presenter: content/presenters/luke-parker.mdx
+    otherEvent:
+      title: 1-Day Clean Architecture Superpowers Tour.
+      eventURL: >-
+        https://www.ssw.com.au/ssw/Events/Training/Clean-Architecture-Superpowers-Tour.aspx
+    _template: PresenterBlock
+  - prose: true
+    centerAlignedText: true
```

Comment:
Should this be centre aligned? I tried it with left align on mine and I thought it looked better - but up to you

Reply:
the issue is with the title which always remains in the center.

Reply:
Yeah, that looks OK I reckon because its a heading - but up to you